src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -10,9 +10,9 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 import Since from '~/components/Since.astro';
 import ReadMore from '~/components/ReadMore.astro'
 
-This adapter allows Astro to deploy your [`hybrid` or `server` rendered site](/en/basics/rendering-modes/#on-demand-rendered) to [Vercel](https://www.vercel.com/).
+This adapter allows Astro to deploy your [`static`, `hybrid` or `server` rendered site](/en/basics/rendering-modes/#on-demand-rendered) to [Vercel](https://www.vercel.com/) and configure additional services such as [Vercel Web Analytics](https://vercel.com/docs/analytics).
```

Comment:
```suggestion
This adapter allows Astro to deploy your [`hybrid` or `server` rendered site](/en/basics/rendering-modes/#on-demand-rendered) to [Vercel](https://www.vercel.com/).
```
Since adapters are not needed for `static` sites *in general*, I think it's more helpful if this statement stays as is, and we instead explain further below where we do talk about static sites!